docker/run_local_docker.sh

```diff
@@ -55,20 +55,21 @@ else
     echo "Skipping Maven Build"
 fi
 
-cd docker/local-metadata || exit
+#cd docker/local-metadata || exit
 
 echo "Stopping any previous Local Docker Containers"
-docker compose -f docker-compose-postgres.yml down
+docker compose  -f docker/local-metadata/docker-compose-postgres.yml down
 docker compose down
-
+rm -rf $PWD/docker-volume
```

Comment:
Add a condition to run only if the `docker-volume` directory exists!

Reply:
done